Library/ViewModels/ProjectPamphletViewModelTests.swift

```diff
@@ -460,43 +460,7 @@ final class ProjectPamphletViewModelTests: TestCase {
     }
   }
 
-  func testGoToDeprecatedManagePledge_ManagingPledge_featureNativeCheckoutPledgeView_Disabled() {
```

Comment:
Did this get missed when we removed the old checkout code, or does this PR need to be updated with the latest changes from master?

Reply:
I'm updating this PR with the changes in `featured-errored-pledges`. `featured-errored-pledges` should be updated with the latest changes from `master`.